sui/operators.js

```diff
@@ -1,79 +1,44 @@
 const { Command, Option } = require('commander');
 const { Transaction } = require('@mysten/sui/transactions');
-const { bcs } = require('@mysten/sui/bcs');
-const { printInfo, loadConfig } = require('../common/utils');
-const { operatorsStruct } = require('./types-utils');
+const { printInfo, printError, loadConfig } = require('../common/utils');
 const { addBaseOptions, addOptionsToCommands, parseSuiUnitAmount } = require('./cli-utils');
 const { getWallet, printWalletInfo, broadcast } = require('./sign-utils');
-const { getBcsBytesByObjectId, findOwnedObjectId } = require('./utils');
+const { findOwnedObjectId } = require('./utils');
 
-async function getGasCollectorCapId(client, gasServiceConfig, contractConfig) {
+function operatorMoveCall(contractConfig, gasServiceConfig, operatorCapId, moveCall) {
     const operatorId = contractConfig.objects.Operators;
+    const gasCollectorCapId = gasServiceConfig.objects.GasCollectorCap;
 
-    // Get and parse operator data
-    const operatorBytes = await getBcsBytesByObjectId(client, operatorId);
-    const parsedOperator = operatorsStruct.parse(operatorBytes);
-
-    // Get the capabilities bag ID
-    const bagId = parsedOperator.caps.id;
+    const tx = new Transaction();
```

Comment:
Modify an existing tx instead? Makes it easier to add batching for operator calls if we want later